tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraEncryptedEmailFastViewGetMethodContract.scala

```diff
@@ -468,6 +469,29 @@ trait LinagoraEncryptedEmailFastViewGetMethodContract {
       .isEqualTo(MESSAGE_PREVIEW)
   }
 
+  @Test
```

Comment:
Should we write a separate contract class for this?
Then we can extends it in both 
`LinagoraEncryptedEmailFastViewGetMethodContract.scala`,
`LinagoraEncryptedEmailDetailedViewGetMethodContract.scala`

Reply:
Mmm IMO no.

Separate contract is IMO overkill, duplication is IMO not needed.